docs/source/author/chapters.rst

```diff
@@ -33,7 +31,7 @@ The ``content`` directory for the chapters application contains
 Content Files
```

Comment:
Under `- a directory for each language in which content exists, named using the Django locale code. This directory contains`:

- 'contains' needs a colon

For the following I'm assuming this is about the actual development directory structure, rather than anything generated by django itself:

- Perhaps say `Content Markdown files, arranged into folders by language`

> `- YAML files containing translatable strings`

- This is false, the yaml files dictate structure, in the format talked about in `Understanding configuration files`